lib/saluki-io/src/net/addr.rs

```diff
@@ -178,3 +181,9 @@ impl From<ProcessCredentials> for ConnectionAddress {
         Self::ProcessLike(Some(creds))
     }
 }
+
+impl<'a> Connected<&'a Connection> for ConnectionAddress {
+    fn connect_info(target: &'a Connection) -> Self {
+        target.remote_addr()
+    }
+}
```

Comment:
We had to add this so that we can use our listener stuff from `saluki-io` with `axum`.